vitrina/settings.py

```diff
@@ -199,7 +199,7 @@
 USE_I18N = True
 USE_L10N = True
 
-USE_TZ = True
+USE_TZ = False
```

Comment:
We do want to use timezones, please return this back to `True` and fix datetime awarnes of timezone instead.